lib/matplotlib/__init__.pyi

```diff
@@ -70,14 +70,22 @@ def get_cachedir() -> str: ...
 def get_data_path() -> str: ...
 def matplotlib_fname() -> str: ...
 
-class RcParams(dict[str, Any]):
+class RcParams(MutableMapping):
     validate: dict[str, Callable]
+    namespaces: tuple
+    single_key_set: set
     def __init__(self, *args, **kwargs) -> None: ...
+    def _split_key(self, key: str, sep: str = ".") -> tuple[list, int]: ...
```

Comment:
```suggestion
    def _split_key(self, key: str, sep: str = ...) -> tuple[list, int]: ...
```
In stub files, the standard practice is to only use `...` for default values, indicating it is optional, but not actually providing the value (not fully sure why, but that is standard practice)

I've also not necessarily been type hinting private methods, though that was more for putting some reins on the first pass than a "we shouldn't type hint private things"